storybook/RelativeFontProvider.js

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Do we need a test file with tests for this?

Reply:
Not really, it's part of the example app/storybook, we don't test or publish any of this. If it's not working will be pretty clear anyway

Reply:
In which case all is good from me!